core/jvm/src/main/scala/cats/effect/unsafe/HelperThread.scala

```diff
@@ -58,6 +58,7 @@ import java.util.concurrent.atomic.{AtomicBoolean, AtomicInteger}
 private[effect] final class HelperThread(
     private[this] val threadPrefix: String,
     private[this] val blockingThreadCounter: AtomicInteger,
```

Comment:
⛏️ Because we use an "increment, run, decrement" approach, I might consider renaming this;
```suggestion
    private[this] val blockingThreadGauge: AtomicInteger,
```

Reply:
Feel free to tackle this in the code.